views/contact_information.py

```diff
@@ -0,0 +1,27 @@
+from rest_framework import generics, permissions
+from rest_framework.exceptions import NotFound
+from formula_one.serializers.generics.contact_information import ContactInformationSerializer
+from omniport.utils.switcher import load_serializer
+
+
+class ContactInformationView(generics.RetrieveUpdateAPIView):
+    """
+    View for RU operations on contact information
+    """
+
+    permission_classes = [permissions.IsAuthenticated, ]
+    serializer_class = ContactInformationSerializer
+
+    def get_object(self):
+        """
+        Return the contact information of the person currently logged in
+        :return: the contact information of the person currently logged in
+        """
+
+        person = self.request.person
+        try:
+            return person.contact_information.get()
```

Comment:
This will throw an exception if more than one instance is returned. Maybe we need to add something to handle the error? @pradumangoyal

Reply:
Actually, we use `.first()` at every place, so it maintains consistency as well as doesn't throw an error for None or multiple instances.